packages/react-mutation-mapper/src/component/track/Track.tsx

```diff
@@ -110,25 +110,46 @@ export default class Track extends React.Component<TrackProps, {}> {
     }
 
     @action.bound
-    onTrackCircleClick(circleComponent: TrackCircle) {
+    onTrackCircleClick(circleComponent: TrackItem) {
         updatePositionSelectionFilters(
             this.props.dataStore,
-            circleComponent.props.spec.codon,
+            circleComponent.props.spec.startCodon,
             this.shiftPressed,
             this.props.defaultFilters
         );
     }
 
     @action.bound
-    onTrackCircleHover(circleComponent: TrackCircle) {
+    onTrackRectClick(rectComponent: TrackItem) {
```

Comment:
We should unite `onTrackRectClick` with `onTrackCircleClick` and `onTrackRectHover` with `onTrackCircleHover`. They essentially do the same thing. Upon hover highlight the positions within range, on click select the positions within range. In case of a circle the range is 1.